frontend/src/metabase/query_builder/components/filters/modals/BulkFilterList/BulkFilterList.tsx

```diff
@@ -95,20 +100,23 @@ const BulkFilterListItem = ({
   }, [filters, dimension]);
 
   return (
-    <ListRow>
-      <ListRowLabel data-testid="dimension-filter-label">
-        {dimension.displayName()}
-      </ListRowLabel>
+    <ListRow
+      aria-label={`filter-field-${dimension.displayName()}`}
+      data-testid="dimension-filter-row"
+    >
```

Comment:
put a label on the whole row to make selecting and testing easier now that the operator selector and value input are in different components

Reply:
nit: aria-label is expected to be a human-readable string. Can we adjust it and use `testid` in tests, if needed?